packages/atomic/cypress/integration/breadbrumb-manager.cypress.ts

```diff
@@ -122,17 +113,28 @@ describe.skip('Breadcrumb Manager Test Suites', () => {
 
   describe('Custom properties test ()', () => {
     beforeEach(() => {
-      setupComponents('collapse-threshold=4 category-divider=";"');
+      setupComponents('collapse-threshold=3 category-divider=";"');
       createAliasFacetUL(numericFacetProp.field, FacetSelectors.numericFacet);
     });
 
-    it('should hide 3 fields when 7 are selected', () => {
-      for (let n = 0; n < 7; n++) {
+    it.skip('should hide 2 fields when 5 are selected', () => {
+      for (let n = 0; n < 5; n++) {
         clickOnNthFacet(n);
         cy.wait(150);
       }
       createBreadcrumbShadowAlias();
-      cy.get(BreadcrumbAlias.breadcrumbs).should('have.length', 4);
+      cy.get(BreadcrumbAlias.breadcrumbs).should('have.length', 3);
+    });
+
+    it('should display all fields on clicking "Show More"', () => {
```

Comment:
Does this test description match what the test is doing? It seems like the test is clicking 5 facet values and asserting that here are 5 breadcrumbs?

How is it related to the facet "show more"?

Reply:
That's right. I'll change the description to make it more clear.